core/drivers/fabric-driver/makefile

```diff
@@ -18,28 +18,28 @@ build-local:
 		cp -r ../../../common/protos-js/peer ./protos-js/ && \
 		cp -r ../../../common/protos-js/relay ./protos-js/ && \
 		cp -r ../../../common/protos-js/package.json ./protos-js/
```

Comment:
Should we add `identity` here too just for completion? May be needed at a later date.

Reply:
Done!